src/main/java/org/wso2/siddhi/plugins/idea/completion/SiddhiCompletionUtils.java

```diff
@@ -0,0 +1,997 @@
+                } else {
+                    EditorModificationUtil.moveCaretRelatively(editor, 1);
+                }
+            }
+        };
+    }
+
+    /**
+     * Creates a <b>Type</b> lookup element.
+     *
+     * @param name          of the lookup
+     * @param insertHandler insert handler of the lookup
+     * @return {@link LookupElementBuilder} which will be used to create the lookup element.
+     */
+    @Nonnull
+    private static LookupElementBuilder createDataTypeLookupElement(@Nonnull String name,
+                                                                    @Nullable InsertHandler<LookupElement>
+                                                                            insertHandler) {
+        return createLookupElement(name, insertHandler).withTypeText("Data Type");
+    }
+
+    /**
+     * Creates a <b>Type</b> lookup element.
+     *
+     * @param name          of the lookup
+     * @param insertHandler insert handler of the lookup
+     * @return {@link LookupElementBuilder} which will be used to create the lookup element.
+     */
+    @Nonnull
+    private static LookupElementBuilder createWindowProcessorTypeLookupElement(@Nonnull String name,
+                                                                               @Nullable InsertHandler<LookupElement>
+                                                                                       insertHandler) {
+        return createLookupElement(name, insertHandler).withTypeText("Window Processor");
+    }
+
+    /**
+     * Creates a <b>Type</b> lookup element.
+     *
+     * @param name          of the lookup
+     * @param insertHandler insert handler of the lookup
+     * @return {@link LookupElementBuilder} which will be used to create the lookup element.
+     */
+    @Nonnull
+    private static LookupElementBuilder createDefineSnippetTypeLookupElement(@Nonnull String name,
+                                                                             @Nullable InsertHandler<LookupElement>
+                                                                                     insertHandler) {
+        return createLookupElement(name, insertHandler).withTypeText("Snippet");
+    }
+
+    /**
+     * Creates a <b>Type</b> lookup element with a user given Type Text.
+     *
+     * @param name          of the lookup
+     * @param insertHandler insert handler of the lookup
+     * @param text          string which needed to be shown as type text
+     * @return {@link LookupElementBuilder} which will be used to create the lookup element.
+     */
+    @Nonnull
+    private static LookupElementBuilder createLookupElementWithCustomTypeText(@Nonnull String name,
+                                                                              @Nullable InsertHandler<LookupElement>
+                                                                                      insertHandler, @Nonnull String
```

Comment:
Move the string parameter declaration to a new line.

Reply:
Fixed in commit 3aa87a6